domain-server/src/DomainServerSettingsManager.cpp

```diff
@@ -863,6 +863,18 @@ void DomainServerSettingsManager::processNodeKickRequestPacket(QSharedPointer<Re
         // pull the UUID being kicked from the packet
         QUuid nodeUUID = QUuid::fromRfc4122(message->readWithoutCopy(NUM_BYTES_RFC4122_UUID));
 
+        bool hasOptionalBanParameters = false;
+        bool banByUsername = false;
+        bool banByFingerprint = false;
+        bool banByIP = false;
+        // pull optional ban parameters from the packet
+        if (message.data()->getSize() > NUM_BYTES_RFC4122_UUID) {
```

Comment:
I can't think of an alternative because I assume you're trying to avoid bumping the protocol, but the way this works by basing it on the size of the packet seems kinda fragile.  the next time we bump the protocol, could we also remove this check and just have the optional parameters become standard?

Reply:
Added issue: https://github.com/vircadia/vircadia/issues/1093

Reply:
maybe we should check that this equals the exact size we expect instead of it being larger than the old size.  this would prevent someone from sending a malformed packet and crashing the mixer